imu_calibration:
  acc_offset: [0.725690960884, 0.173638924956, -0.06346839905]
  gyro_offset: [0.473282456398, -1.19083964825, -3.98473286629]
  gryo_scale: [0.01744444444, 0.01744444444, 0.01744444444]
  acc_scale: [1, 1, -1]